media.go

```diff
@@ -195,9 +195,24 @@ type Location struct {
 	// Longitude
 	Lng float32 `json:"longitude"`
 
+	// Horizontal Accuracy
+	HA *float32 `json:"horizontal_accuracy,omitempty"`
```

Comment:
In this particular case, it's honestly better not to abbreviate words, _horizontal accuracy_ isn't something that can be recognized by these two letters.